server/lib/imageproxy.ts

```diff
@@ -315,6 +329,31 @@ class ImageProxy {
   private getCacheDirectory() {
     return path.join(baseCacheDirectory, this.key);
   }
+
+  private getMimeExtension(mime: string): string {
+    switch (mime) {
+      case 'image/png':
+        return 'png';
+      case 'image/jpeg':
+        return 'jpeg';
+      case 'image/jpg':
+        return 'jpg';
+      case 'image/apng':
+        return 'apng';
+      case 'image/gif':
+        return 'gif';
+      case 'image/vnd.microsoft.icon':
+        return 'ico';
+      case 'image/tiff':
+        return 'tiff';
+      case 'image/webp':
+        return 'webp';
+      default:
+        break;
+    }
+
+    return '';
+  }
```

Comment:
I don't have a strong opinion on this, but shouldn't we use a library to do this conversion (like [this one](https://github.com/broofa/mime?tab=readme-ov-file#quick-start))?
I feel like this is an easy way to mess with the [official specification](https://www.iana.org/assignments/media-types/media-types.xhtml#image).

@Fallenbagel any opinion on this?

Reply:
I agree

Reply:
My bad, because I 100% agree. I was going to ask whether using that library for this situation was acceptable and forgot to do it and left it as it was.